src/modules/data/entities/aggregate/global/global.character.aggregate.entity.ts

```diff
@@ -6,7 +6,7 @@ import {World, worldArray} from '../../../constants/world.consts';
     name: 'aggregate_global_characters',
 })
 @Index(['character', 'world'], {unique: true})
-export default class Character {
+export default class GlobalCharacterAggregateEntity {
```

Comment:
I would really recommend smaller names for readability. Besides such a detailed description of the class is really unnecessary as the place they will be used will not have any conflicting class naming anyway.

When it comes to classnaming when the name becomes too long put them in separate folders.

Reply:
While I agree in terms of folder structure, when it comes to code it's very nice to see exactly what you're dealing with, rather than having to look at the import to see actually which class you're dealing with. In addition, if you start sharing class names, you will hit import errors.

Consider these examples:

### Example 1 - Generic class names
```
import ClassAggregate from '/controllers/global/class.aggregate.ts`

class SomeClass {
    private someFunction() {
        ... blah
        ClassAggregate.doAThing();
    }
}
```
Issue I have here is unless you look at the import, which class aggregate are you working with? Instance? Global? What about if you want to use both? We'd have to do import renaming which is really fudgy.

### Example 2 - Specific class names
```
import GlobalClassAggregate from '/controllers/global/global.class.aggregate.ts`
import InstanceClassAggregate from '/controllers/instance/instance.class.aggregate.ts`

class SomeClass {
    private someFunction() {
        ... blah
        GlobalClassAggregate.doAThing();
        InstanceClassAggregate.doAThing();
    }
}
```
This in my opinion is much cleaner, as you're able to see without having to look at the imports exactly which class you're dealing with, it's specific and there's no chance for errors, for the sake of a few extra characters.